EllisWeb.Gematria/Calculator.cs

```diff
@@ -168,49 +168,110 @@ public static long GetNumericGematriaValue(string sourceString, GematriaType gem
         /// </remarks>
         /// <returns>Gemtria Numeric representation of given number</returns>
         public static string ConvertToGematriaNumericString(long number, bool includeSeparators = true, char thousandsSeparator = '\'', char tensSeparator = '"')
```

Comment:
please decorate with an `[obsolete]` attribute, and give explanation for this in `remarks`